logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageRepository.kt

```diff
@@ -211,6 +215,14 @@ class MessageDataSource(
         )
     }
 
+    override suspend fun persistSystemMessageToAllConversations(
```

Comment:
Let's add unit test checking the message really inserted into all conversations?